spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Sequential.scala

```diff
@@ -30,61 +30,91 @@ import scala.reflect.ClassTag
 class Sequential[T: ClassTag]
 (implicit ev: TensorNumeric[T]) extends Container[Activity, Activity, T] {
 
+  private var evaluateOnly = false
+
+  /**
+   * If evaluateOnly = true, then this sequential only works in evaluate mode, and not work in
```

Comment:
If evaluateOnly = true, when in training, it will not do layer forward (but do output = input, for layers after that), and will not do layer backward (but do gradInput = gradOutput, for layers before that)